plugins/anx-udev/anx-udev.quirk

```diff
@@ -0,0 +1,12 @@
+#Analogix device
+
+# Analogix usb device
+[DeviceInstanceId=USB\VID_1F29&PID_7518]
```

Comment:
You don't need the `DeviceInstanceId=` prefix with master anymore.

Reply:
Hi Hghsie,
So just remove DeviceInstanceId? Like below:
[USB\VID_1F29&PID_7518]
Thank you!

Reply:
Exactly that, thanks.